k8s/models/apiextensions_v1_custom_resource_definition.py

```diff
@@ -77,6 +77,31 @@ class JSONSchemaProps(Model):
     x_kubernetes_preserve_unknown_fields = Field(bool, name="x-kubernetes-preserve-unknown-fields")
 
 
+class JSONSchemaPropsStatusEnabled(Model):
```

Comment:
Fields with type `six.text_type` should be changed to `str`, after dropping Python 2.x support and removing dependency on six in #116